code/script.js

```diff
@@ -35,17 +42,52 @@ const showMessage = (message, sender) => {
 
 // Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
+  showMessage("Welcome to report error messages about our municipal streets! Do you want to make a report?", 'bot');
+};
```

Comment:
This function has been defined, but not invoked anywhere. That is why the chatbot does not show I guess. You might have accidentally removed it from this file. You could use `setTimeout(greetUser, 1000)` to make the first message appear one second after the website is loaded.